### ISSUE TITLE
DynamicPageTitle: a11y - When tabbed into the "..." toggle button, it should set aria-expanded aria-haspopup attributes

### ISSUE THREAD

**Issue by gurkirpalgill** [SAP/ui5-webcomponents-react#2613]
**Describe the bug**
When user tabs into the toggle button "..." in DynamicPageTitle actions menu, it should set aria-expanded and aria-haspopup attributes appropriately.

**Screenshots**
<img width="1879" alt="Screen Shot 2022-02-17 at 5 51 53 PM" src="https://user-images.githubusercontent.com/65705637/154603377-d81a1260-2d85-4b75-ac02-7727c3c075db.png">

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 1.1.0
`@ui5/webcomponents-react` version: 0.21.0
Operating System: Mac OS
Browser: Chrome

.

**Comment by Lukas742** [comment]
Hi @gurkirpalgill 

the `ToggleButton` which is used as the "Show More" button is not yet supporting the `accessibilityAttributes`, therefore we cannot enhance the button in the shadow root with these a11y attributes. However, the ui5 Web Components team has already [implemented this behavior](https://github.com/SAP/ui5-webcomponents/pull/4655), so now we just have to wait for the next release to use it.

**Comment by gurkirpalgill** [comment]
Please re-open the issue  as it is partially resolved. aria-expanded and aria-haspopup attributes are set now, but the value of aria-expanded is not set to "false" when user has navigated to the "..." menu and presses escape. Menu collapses on pressing escape, but the aria-expanded still shows "true" instead of "false".
<img width="1910" alt="Screen Shot 2022-03-04 at 2 28 53 PM" src="https://user-images.githubusercontent.com/65705637/156851979-39bf1cd1-8a58-448e-b7ae-19e19da03645.png">

<img width="1913" alt="Screen Shot 2022-03-04 at 2 44 20 PM" src="https://user-images.githubusercontent.com/65705637/156852302-2d85b6e3-396b-424e-83dc-612d9748eac8.png">
